server/routes/api/roles.js

```diff
@@ -57,4 +58,13 @@ function updateRole (req, res) {
     });
 }
 
+async function deleteRole (req, res, next) {
+    try {
+        await req.$role.remove();
```

Comment:
Role should be removed from users somewhere around here

Reply:
I agree, fixed